packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts

```diff
@@ -251,6 +251,11 @@ export class CliArgsHandlerBase implements ICliArgsHandler {
     if (args.unionDefaultGraph) {
       context[KeysQueryOperation.unionDefaultGraph.name] = true;
     }
+
+    // Define union default graph
```

Comment:
Wrong jsdoc.

Reply:
Option also needs to be added here: https://github.com/comunica/comunica/blob/master/packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts#L66

Reply:
And also here: https://github.com/comunica/comunica/blob/master/packages/actor-init-query/lib/ActorInitQueryBase.ts#L124